openbb_platform/openbb/assets/reference.json

```diff
@@ -33,6 +33,7 @@
                 "sec@1.3.4",
                 "tiingo@1.3.4",
                 "tradingeconomics@1.3.4",
+                "us_eia@1.0.0b0",
```

Comment:
Are we including this into the main distribution or an extra? I'm curious because of the deps it pulls with it (the excel, and the new cache). Have you checked if those deps are pure python or with extensions that require some sort of build tools to be installed?

Reply:
The intention is to include it as part of the main distribution because it is public domain data, free to access, and provides coverage of widely followed energy commodities fundamentals. If there is a reason not to, let's discuss!

As finance is heavily reliant on Excel/spreadsheet files, I/O packages related to handling these types of files feels like it should ship with the main distribution. I believe the packages being introduced are "lightweight" additions that do not introduce anything that requires special handling or installation considerations - ie, nothing like pyarrow, numba, optimized, or uncompiled binaries.

Maybe I'm missing something, so let me know if there is anything overlooked while arriving at the conclusion.

- openpyxl is the Pandas extra that enables the ExcelFile class and IO of Excel files in general.
  - XLRD is for handling the older `.xls` file format, and every Python Excel library actually uses this under the hood for handling this format. XLS files are still widely distributed.
  - The XML components are already present in the main distribution.


async-lru is a simple extension that relies only on core Python elements. The library is just an `__init__.py`, and it imports:

```
import asyncio
import dataclasses
import sys
from asyncio.coroutines import _is_coroutine  # type: ignore[attr-defined]
from functools import _CacheInfo, _make_key, partial, partialmethod
from typing import (
    Any,
    Callable,
    Coroutine,
    Generic,
    Hashable,
    Optional,
    OrderedDict,
    Set,
    Type,
    TypedDict,
    TypeVar,
    Union,
    cast,
    final,
    overload,
)
```

Reply:
Thanks for this reply. I helped a lot to understand the async-lru part and i've checked the openpyxl and it's a pure python package. I identified that by looking at the lock file and seeing that the package is platform agnostic

<img width="1077" alt="image" src="https://github.com/user-attachments/assets/d1eb9967-4aaf-4e85-bff3-8528781d1e88">